synapse/models/infotech.py

```diff
@@ -934,6 +956,103 @@ def getModelDefs(self):
                         'doc': 'An array of ChildOf CWE Relationships.'
                     }),
                 )),
+
+                ('it:sec:metrics', {}, (
```

Comment:
Do we need to track when mitigations arent available for a vuln?

Reply:
That would just be an unmitigated result. There are often ways to mitigate bugs even without patches. We can discuss further in the future but not necessary for today.